### DISCUSSION TITLE
Bat theme looks darker in ranger than in terminal output

### DISCUSSION THREAD

**Discussion by 3rfaan** [sharkdp/bat#2440] [Q&A]
I've made a custom theme for bat inspired by the Everforest theme by sainnhe. My issue is that in ranger the colors look darker than in the normal output in the terminal. I've attached a picture to show you what I mean.

Above you can see a normal bat output and it shows the right colors. But below in ranger when I open the exact same file in the preview the colors appear to be darker.

Is there any idea on how to get the true colors into the ranger preview?

![screenshot-20230103-182139Z-all](https://user-images.githubusercontent.com/28968837/210418529-67fa1f59-f033-4faf-a6da-b9697b6bf655.png)

**Comment by keith-hall** [comment]
I have no idea, but it might be worth trying to let ranger know how many colors your terminal supports: https://github.com/ranger/ranger/issues/547#issuecomment-219956667
